tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Update Party7171.feature

```diff
@@ -0,0 +1,85 @@
+﻿Feature: Updating the Ordering Party in a Buyer Section
+    As a Buyer
+    I want to update an Ordering Party
+    So that I can ensure the information is correct
+
+Background:
+    Given Addresses exist
+        | Line1 | Line2      | Line3      | Line4          | Line5          | Town      | County  | Postcode | Country        |
+        | 4     | Upper Flat | Rocks Lane | Little Village | Bigger Village | Some Town | W Yorks | LS 1 3AP | United Kingdom |
+    Given Contacts exist
+        | FirstName | LastName | EmailAddress            | TelephoneNumber |
+        | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
+    Given Orders exist
+        | OrderId    | OrganisationId                       | OrganisationName | OrganisationOdsCode | OrganisationAddressPostcode | OrganisationContactEmail | Description      | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        |
+        | C000014-01 | 4af62b99-638c-4247-875e-965239cd0c48 | Hampshire CC     | 432432              | LS 1 3AP                    | Fred.robinson@email.com  | Some Description | 1             | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@7171
+Scenario: 1. Update an ordering party primary contact results in a database update 
+    When the user makes a request to update the order party on the order with the ID C000014-01

```

Comment:
You should be teesting all the parts together as one object for a single update. I can push changes to a branch, as I've started to do that for my integration tests. But they should test the whole object, not parts